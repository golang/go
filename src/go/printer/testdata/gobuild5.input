//go:build !(x || y) && z
// +build something else

package p
