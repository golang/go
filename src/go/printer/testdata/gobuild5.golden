//go:build !(x || y) && z
// +build !x,!y,z

package p
